### ISSUE TITLE
govc device.info not return disk size when govc creates VM against vcsim

### ISSUE THREAD

**Issue by leslie-qiwa** [vmware/govmomi#1108]
I can use `govc vm.create -m 2048 -c 2  -disk.controller ide -disk 1GB -k -pool=/DC0/host/DC0_C0/Resources -net=/DC0/network/VM\ Network disk-test-vm` to create VM against vcsim, but when I run `govc device`, it doesn't return disk size in device summary, which is same as label.

```
Name:            ide-202                                           
  Type:          VirtualIDEController                              
  Label:         ide-202                                           
  Summary:       ide-202                                           
  Key:           202                                               
  Devices:                                                         
Name:            disk--201-0                                       
  Type:          VirtualDisk                                       
  Label:         disk--201-0                                       
  Summary:       disk--201-0                                       
  Key:           203                                               
  File:          [LocalDS_0] disk-test-vm/disk-test-vm.vmdk        
```

The output from real ESX host is like 
```
Name:               disk-1000-0
  Type:             VirtualDisk
  Label:            Hard disk 1
  Summary:          16,777,216 KB
  Key:              2000
  Controller:       lsilogic-1000
  Unit number:      0
  File:             [nvme0] puppy1/puppy1_0.vmdk
```

**Comment by dougm** [comment]
@leslie-qiwa is #1110 good enough or do you need the commas?

**Comment by leslie-qiwa** [comment]
It is more accurate. Thanks for confirming!

**Comment by leslie-qiwa** [comment]
Sorry for not very precisely. I probably need commas because it is more align with above output from real ESX host.
Not sure if govmomi has library for such conversation. Here is what I made, https://play.golang.org/p/tn7YdWtd6O2. If it is ok, I can submit one PR.

**Comment by dougm** [comment]
@leslie-qiwa ok sure, a PR from you would be great.  There isn't anything in govmomi to provide that formatting, I don't think Go stdlib has anything either.  Let's use your function, it can be unexported (`numberToString`) in the simulator package for now.

**Comment by dougm** [comment]
Closed by #1115
